File: Iniciante/1074 - Par ou Ímpar/JavaScript/1074.js
function main() {
  const input = require("fs").readFileSync(0, "utf8").trim().split("\n");
  let index = 0;
  const amount = parseInt(input[index++]);

  for (let i = 0; i < amount; i++) {
    const value = parseInt(input[index++]);

    if (value === 0) {
      console.log("NULL");
    } else {
      const parity = value % 2 !== 0 ? "ODD" : "EVEN";
      const sign = value > 0 ? "POSITIVE" : "NEGATIVE";
      console.log(`${parity} ${sign}`);
    }
  }
}

main();

File: Iniciante/1074 - Par ou Ímpar/Python/1074.py
def main():
    amount = int(input())

    for _ in range(amount):
        value = int(input())

        if value == 0:
            print("NULL")
        else:
            parity = "ODD" if value % 2 != 0 else "EVEN"
            sign = "POSITIVE" if value > 0 else "NEGATIVE"
            print(f"{parity} {sign}")


if __name__ == "__main__":
    main()
